src/Content/Text/BBCode.php

```diff
@@ -2243,7 +2243,7 @@ public static function getTags($string)
 			$string = str_replace(['[', ']'], ["\n[", "]\n"], $string);
 
 			// ignore anything in a bbtag
-			$string = preg_replace('/\[(.*?)\]/sm', '', $string);
+			$string = preg_replace('/\[(.*?)\]/gsm', '', $string);
```

Comment:
There's no need for the `g` flag in `preg_replace` specifically since by default it will replace all matches by the replacement.

PHP raises an error if you mention it: `preg_replace(): Unknown modifier 'g'`

And this error is making the tests fail.